app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AttachmentsController < ApplicationController
+  def show
+    @attachment ||= Attachment.find(params[:id])
```

Comment:
oh, memoization is used only when code calls several times - here it's useless

Reply:
thanks, fixed